components/DownloadButton.tsx

```diff
@@ -11,35 +13,43 @@ import StatusIcon from "./StatusIcon"
 import Tooltip from "./Tooltip"
 
 type DownloadButtonProps = {
-  proof: Proof & { team?: Team }
+  proof: Proof
   className?: string
 }
 
 const DownloadButton = ({ className, proof }: DownloadButtonProps) => {
-  const { proof_status, proof: binary, team } = proof
+  const { proof_status, proof_id, proof: binary, size, team } = proof
+
   const teamName = team?.team_name ? team.team_name : "Team"
 
   const sizingClassName =
     "h-8 gap-2 self-center text-2xl sm:max-md:w-40 lg:w-40"
-  const labelClassName =
-    "hidden text-nowrap text-xs font-bold sm:block md:hidden lg:block"
+  const labelDisplay = "hidden sm:inline-block md:hidden lg:inline-block"
+  const labelStyle = "text-nowrap text-xs font-bold"
+
   const fakeButtonClassName =
     "bg-body-secondary/10 hover:bg-body-secondary/10 cursor-auto"
 
   if (proof_status === "proved")
     return (
-      <Button
-        variant="outline"
-        className={cn(sizingClassName, className)}
-        size="icon"
-        disabled={!binary}
-        asChild
-      >
-        <a href={`/api/v0/proofs/download/${proof.proof_id}`} download>
-          <ArrowDown />
-          <span className={labelClassName}>Download proof</span>
-        </a>
-      </Button>
+      <div className="flex flex-col items-center">
+        <Button
+          variant="outline"
+          className={cn(sizingClassName, className)}
+          disabled={!binary}
+          asChild
+        >
+          <a href={`/api/v0/proofs/download/${proof_id}`} download>
+            <ArrowDown />
+            <span className={labelStyle}>
+              <span className={labelDisplay}>
+                Download {size ? "" : "proof"}
```

Comment:
@nloureiro opinion here?

In current implementation, the download label for proofs of 5kb and 10mb would read:

`Download 5kB proof`
`Download 10MB proof`

I think having the size outside of the main text is the most normal pattern to use here.

`Download proof (5kB)`
`Download proof (10MB)`

Reply:
yes, this is better
```
Download proof (5kB)
Download proof (10MB)
```

Reply:
Ok. All buttons will get bigger if we do this approach

Reply:
...at least that was my thought process in this. These are laid in a column, so there is a shared width. Hence trying to _replace_ "proof" with the size to keep about the same width, but I'm happy to try and squeeze it all in.

Reply:
Could also consider two rows:

<img width="1106" alt="image" src="https://github.com/user-attachments/assets/15a81ec4-0db1-489e-be68-ced9d7b9941b" />

Just ideating cc: @nloureiro

Reply:
<img width="1492" alt="image" src="https://github.com/user-attachments/assets/f96c1034-418a-4b92-b8bc-0033ed22eb8b" />

This is the desktop appearance with everything on one line (above suggestion)

...and initial PR suggestion for reference:

<img width="1442" alt="image" src="https://github.com/user-attachments/assets/ad99dc3d-9df7-4d9d-af5d-00d663cf84dc" />